content/helpers_test.go

```diff
@@ -81,6 +151,12 @@ func TestCopy(t *testing.T) {
 				testcase.source.size,
 				testcase.source.digest)
 
+			// if an error is expected then further comparisons are not required
```

Comment:
@mikebrow There was one merge conflict while cherry-picking here in the tests. Other than that it was clean.